crypto/time.go

```diff
@@ -7,7 +7,9 @@ import (
 
 // UpdateTime updates cached time.
 func UpdateTime(newTime int64) {
-	pgp.latestServerTime = newTime
+	if(newTime > pgp.latestServerTime) {
```

Comment:
```suggestion
	if (newTime > pgp.latestServerTime) {
```
The linter complains about this